OpenRA.Mods.Common/Widgets/Logic/MusicPlayerLogic.cs

```diff
@@ -41,12 +41,12 @@ public MusicPlayerLogic(Widget widget, Ruleset modRules, World world, Action onE
 			var playButton = panel.Get<ButtonWidget>("BUTTON_PLAY");
 			playButton.OnClick = Play;
 			playButton.IsDisabled = noMusic;
-			playButton.IsVisible = () => !Sound.MusicPlaying;
+			playButton.IsVisible = () => { !Sound.MusicPlaying && currentSong != null; };
```

Comment:
This should be disabling the buttons (playButton.IsDisabled) instead of hiding them.  As for the actual syntax error, just drop the `{}`.  This change covers part of #8746.